script/Arcueid/Day 8/08_00_ARC08_2_1.txt

```diff
@@ -191,7 +219,8 @@ It has to be.
 [sha:279ab6d3a6e98282f0fae0d263815462e555a9ca]{
 -- Page 130, Offset 19995.
 -- そもそもあの崩落はアルクェイドのヤツが考えなしにやった事だし、吸血鬼事件と北口駅前の土地は関係がない。
-Besides, that collapse happened because Arcueid wasn't thinking. There's no connection between the vampire incident and the land in front of the north gate.
+Besides, that cave-in was the result of Arcueid not thinking things through.
+ There's no connection between the vampire incidents and the land in front of the north gate.
```

Comment:
"vampiric incidents" or "killings". I feel like this phrase is used way too much.

Reply:
yeah it really is, and it's really awkward literally